src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  switch (true) {
+    case days >= LONG_TERM:
+      return (PRICE_PER_DAY * days) - LONG_TERM_DISCOUNT;
+    case days >= SHORT_TERM:
+      return (PRICE_PER_DAY * days) - SHORT_TERM_DISCOUNT;
+    default:
+      return (PRICE_PER_DAY * days);
+  }
```

Comment:
1. It is not a good practice to use `switch` case like this. Usually we pass a variable to test for some cases, but you pass a boolean. Better use `if` here.
2. You are duplicating `(PRICE_PER_DAY * days)`. Create a variable for this calculation.